### DISCUSSION TITLE
cannot import name 'BeanieUserDatabase' from 'fastapi_users.db'

### DISCUSSION THREAD

**Discussion by SwordPL** [fastapi-users/fastapi-users#1414] [Q&A]
## Describe the bug
`ImportError: cannot import name 'BeanieUserDatabase' from 'fastapi_users.db'`
## To Reproduce
```py
>>> from fastapi_users_db_beanie import BeanieBaseUser
>>> from fastapi_users import db
>>> db
<module 'fastapi_users.db' from 'C:\\Users\\user\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\aspect-weaver-backend-IEifFoga-py3.11\\Lib\\site-packages\\fastapi_users\\db\\__init__.py'>
>>> from fastapi_users.db import BeanieUserDatabase, ObjectIDIDMixin
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'BeanieUserDatabase' from 'fastapi_users.db' (C:\Users\user\AppData\Local\pypoetry\Cache\virtualenvs\aspect-weaver-backend-IEifFoga-py3.11\Lib\site-packages\fastapi_users\db\__init__.py)
```

>>>## Expected behavior

The order of operations should not introduce circular dependencies.

## Configuration
 - Python version : 3.11.7
 - FastAPI version : fastapi = "^0.108.0"
 - FastAPI Users version : fastapi-users = {extras = ["beanie", "oauth"], version = "^12.1.2"}

**Comment by DShamuratov** [comment]
probably installation was superficial: **pip install fastapi-users**

try instead reinstalling with this command: **pip install 'fastapi-users[beanie]'**
